CDTDatastore/touchdb/TD_Database.m

```diff
@@ -528,6 +535,8 @@ INSERT INTO INFO (key, value) VALUES ('publicUUID',  '%@')",
             result = NO;
             return;
         }
+
+        *rollback = NO;
```

Comment:
Add comment something like `// Block completed successfully, no need to rollback`